src/Symfony/Component/HttpKernel/Bundle/Bundle.php

```diff
@@ -26,8 +27,10 @@
  *
  * @api
  */
-abstract class Bundle extends ContainerAware implements BundleInterface
+abstract class Bundle implements BundleInterface
```

Comment:
it's not needed because `interface BundleInterface extends ContainerAwareInterface`